dll/cpl/powercfg/lang/pt-PT.rc

```diff
@@ -90,7 +97,7 @@ BEGIN
     ICON IDI_ACDC, IDI_ACDC, 7, 7, 0, 0
     LTEXT "Seleccione as definições de poupança de energia que deseja utilizar.", -1, 37, 6, 208, 20
     GROUPBOX "Opções", -1, 6, 30, 240, 65
-    AUTOCHECKBOX "Mostrar sempre o &icone na barra de tarefas", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "Mostrar sempre o &ícone na barra de tarefas", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
```

Comment:
Is this an "easy" keyboard accelerator to use? (using accentued i)

Reply:
you're right.